packages/shared/ReactElementType.js

```diff
@@ -23,7 +18,4 @@ export type ReactElement = {
 
   // __DEV__
   _store: {validated: boolean, ...},
-  _self: React$Element<any>,
-  _shadowChildren: any,
```

Comment:
Not sure what this was about. Goes way back. https://github.com/facebook/react/pull/7504

Reply:
https://github.com/facebook/react/pull/7001/files#r67037956 :)